doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -143,18 +175,19 @@ the following changes are required:
 
 
 An image built with the above setup creates a box file with the
-extension :file:`.vagrant.libvirt.box`. That box file can now be
-added to vagrant with the command:
+extension :file:`.vagrant.libvirt.box` or
+:file:`.vagrant.virtualbox.box`. That box file can now be added to vagrant
```

Comment:
Maybe make it more explict (to avoid the "can" word. With "can" it sounds a bit too optional):

```
Add the box file to Vagrant with the command:
```